app/templates/partials/summary/final-summary.html

```diff
@@ -0,0 +1,44 @@
+<div class="accordion js-accordion">
```

Comment:
Since we didn't like the term `final-summary` because essentially they are all summaries, can we now not name this `collapsed-summary`, since that is the key we use to display this?